extensions/runtimes/package.json

```diff
@@ -10,6 +10,7 @@
     "watch": "yarn build --watch"
   },
   "dependencies": {
+    "@botframework-composer/types": "file:../../Composer/packages/types",
```

Comment:
Is this how published packages should be expressing dependencies? Not as familiar with file references.

Reply:
This is how the extensions currently reference these libraries that are in a different yarn workspace. I'm not sure if they should be updated to point to the published packages...